js/tetris.js

```diff
@@ -123,26 +146,29 @@ function clearLines() {
 function keyPress( key ) {
     switch ( key ) {
         case 'left':
-            if ( valid( -1 ) ) {
+            if ( valid( -1 ) && !pause ) {
                 --currentX;
             }
             break;
         case 'right':
-            if ( valid( 1 ) ) {
+            if ( valid( 1 ) && !pause ) {
                 ++currentX;
             }
             break;
         case 'down':
-            if ( valid( 0, 1 ) ) {
+            if ( valid( 0, 1 ) && !pause ) {
                 ++currentY;
             }
             break;
         case 'rotate':
             var rotated = rotate( current );
-            if ( valid( 0, 0, rotated ) ) {
+            if ( valid( 0, 0, rotated ) && !pause ) {
```

Comment:
This code can be made simpler by moving the check for a pause key to the beginning of the function before the `switch` is ran. Then you won't need to repeat yourself in each `case`, as you can `return` early.